packages/api/src/controllers/stream.js

```diff
@@ -300,12 +300,20 @@ app.post(
 
     try {
       await req.store.create(doc)
-      trackAction(
-        req.user.id,
-        req.user.email,
-        { name: 'Stream Session Created' },
-        req.config.segmentApiKey,
-      )
+      setImmediate(async () => {
```

Comment:
I think because we're in a `setImmediate` here and we've exited the async loop, any unhandled rejection [will result in the server crashing entirely](https://nodejs.org/api/deprecations.html#DEP0018). No good. Easy enough to work around with a try/catch, feel free to merge after that's added.